src/ImageService.Business/Commands/ImageNews/CreateImageWikiNewsCommand.cs

```diff
@@ -63,14 +63,14 @@ public async Task<OperationResultResponse<CreateImageNewsResponse>> ExecuteAsync
       {
         _httpContextAccessor.HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
 
-        return new OperationResultResponse<CreateImageNewsResponse>
+        return new OperationResultResponse<CreateImageWikiNewsResponse>
         {
           Status = OperationResultStatusType.Failed,
           Errors = errors
         };
       }
 
-      OperationResultResponse<CreateImageNewsResponse> response = new();
+      OperationResultResponse<CreateImageWikiNewsResponse> response = new();
 
       DbImage dbImageNews = _mapper.Map(request);
       DbImage dbPreviewNews = null;
```

Comment:
the same
DbImage dbPreview
or
DbImage dbPreviewWikiNews
and? please, fix all names

Reply:
Done